storage/README.md

```diff
@@ -12,15 +12,19 @@ Getting Started
 ---------------
 
 - Create your project on the [Firebase Console](https://console.firebase.google.com).
-- Copy the initialization snippet into the index.html file by clicking the red HTML
-  logo at the top right of the page with the caption "Add Firebase to your web app".  The snippet which appears should be placed in the `<head>` section of `index.html`.
+- From your Firebase project console, click on red circle button with the caption "Add Firebase to your web app".
+![Alt text](/database/pics/redcircle.png?raw=true "Red Circle")
+- Copy the initialization snippet into the index.html file by clicking the red HTML.
+![Alt text](/database/pics/snippet.png?raw=true "Snippet")
+- The snippet which appears should be placed in the `<head>` section of `index.html`.
 - Enable Anonymous auth in the **Auth > SIGN IN METHOD** tab.
+![Alt text](/database/pics/enable.png?raw=true "Enable auth")
```

Comment:
For this sample developers need to enable "Anonymous" Auth. Could you change the screenshot to show the "Anonymous" identity provider enabled? (currently the screenshot shows the "Password+Login" enabled).